eclipse-scout-core/src/form/fields/LookupBox.ts

```diff
@@ -67,6 +67,16 @@ export abstract class LookupBox<TValue> extends ValueField<TValue[], TValue | TV
 
   protected abstract _initStructure(value: TValue[]);
 
+  protected abstract _createStructureModel(): ModelOf<Widget>;
+
+  override _createChild<T extends Widget>(widgetOrModel: ObjectOrChildModel<T> | string): T {
```

Comment:
This does not work, as it enhances every model of a child with the structureModel. LookupBoxes in scout.rt have two children, the tree/table and the filterBox. With this e.g. the filterBox of a ListBox is enhanced with columns.
For scout.rt this might not be problematic as the properties of a tree/table do not collide with the ones of a RadioButtonGroup, which is normally used as filterBox. But as the type of the filterBox is Widget, it can also be e.g. a Table, and then there will be a lot of unexpected behaviour.